std_msgs/Header header
string joint_name
int16 output                   # GM3508: 电流值, GM6020: 电压值
